ENGAGEHF/Dashboard/ToDo/Views/ToDoRows.swift

```diff
@@ -0,0 +1,48 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+struct ToDoRow: View {
+    @Binding var task: ToDoItem
+    @State var taskComplete = false
+    
+    var body: some View {
+        HStack {
+            Text(task.task)
+            Spacer()
+            Button {
+                taskComplete.toggle()
+            } label: {
+                Image(systemName: taskComplete ? "checkmark.square" : "square")
+            }
+        }
+    }
+}
+
+
+// TODO: Add onDelete functionality / backwards update to record completed task
+struct ToDoRows: View {
+    @Binding var tasks: [ToDoItem]
+    
+    var body: some View {
+        if !tasks.isEmpty {
+            ForEach($tasks, id: \.id) { task in
+                ToDoRow(task: task)
+            }
+        } else {
+            Text("Nothing to do!")
```

Comment:
Similar to the Notifications we can either use off-the-shelf content unavailable views or rather hide the complete section if we don't have any TODOs.

We should identify a logic what TODOs should be there at the beginning when you open the app (we have that in the document) and what TODOs should reset on a daily basis (e.g. take a weight measurement).